styles/colors.css

```diff
@@ -3,7 +3,7 @@
   --light-primary-variant: 223, 193, 19;
   --light-secondary: 69, 109, 203;
   --light-secondary-variant: 89, 135, 216;
-  --light-background: 255, 255, 255;
+  --light-background: 246, 246, 246;
```

Comment:
Can we not change this property now? I think we have to find a point in common about this topic